schemas.py

```diff
@@ -0,0 +1,34 @@
+from pydantic import BaseModel
+from datetime import date
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: str
+    publication_date: date
+
+class BookCreate(BookBase):
+    pass
+
```

Comment:
```suggestion


class BookCreate(BookBase):
    pass


```

Reply:
There should be double empty line before and after class definition. Fix everywhere.